src/java.base/share/classes/jdk/internal/foreign/abi/fallback/LibFallback.java

```diff
@@ -36,14 +36,20 @@ private LibFallback() {}
 
     static final boolean SUPPORTED = tryLoadLibrary();
 
+    @SuppressWarnings("removal")
     private static boolean tryLoadLibrary() {
-        try {
-            System.loadLibrary("fallbackLinker");
-        } catch (UnsatisfiedLinkError ule) {
-            return false;
-        }
-        init();
-        return true;
+        return java.security.AccessController.doPrivileged(
+                new java.security.PrivilegedAction<>() {
```

Comment:
Could use imports instead of using fully qualified names here.

Reply:
No - problem here is that AccessController is deprecated for removal - so if we import that, we'll get a warning on the import.